Jenkinsfile.json

```diff
@@ -1,18 +1,16 @@
 {
-    "container_repo": {
-      "host": "dfsacr.azurecr.io",
-      "master_acct": "k8s-helm",
-      "alt_acct": "k8s-helm",
-      "jenkins_creds_id": "dfsacr_creds",
-      "repo": "k8s-helm",
-      "dockeremail": ".",
-      "dockerfile": "./"
+    "app": {
+        "name": "k8s-helm", 
+        "namespace": "activelink",
```

Comment:
This namespace needs to be changed to utilities. As based on our discussion it is important to separate our essential services.